.github/workflows/biweekly-build-test-and-publish-release.yaml

```diff
@@ -0,0 +1,81 @@
+# Copyright 2024 Dynatrace LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Bi-weekly build, test and publish Release
+
+on:
+  schedule:
+    # At 01:00 UTC on Friday
+    - cron: '0 1 * * 5'
```

Comment:
You can just run it every 14 weeks (`0 1 */14 * *`), starting from a given date. This would simplify your code in `Bi-weekly new tag`.

Reply:
\*every 14 days
I don't think that it's the correct solution: [https://crontab.guru/#0_1_*/14_*_*](https://crontab.guru/#0_1_*/14_*_*)
![image](https://github.com/user-attachments/assets/9f2bc49e-7e24-434c-9838-3537f1a3ab49)
It will finally lose sync with OneAgent releases